multihost_test/Bugzillas/data/bz_667593.sh

```diff
@@ -0,0 +1,16 @@
+function groupPasswordAdd() {
```

Comment:
There's already a similar script in [bz672510_1.sh](https://github.com/RedHatQE/shadow-utils/blob/dd968fc29862b99325e9cf930793e377781e92f5/multihost_test/Bugzillas/data/bz672510_1.sh#L4). Do you think it would be possible to have just one script?

Reply:
These are not same . bz672510_1.sh needs arguments to pass :
execute_cmd(multihost, f"sh /tmp/bz672510_1.sh {tgroup} {tgroup}")

But this one does not need any arguments to pass . Only function name is same not the work

Reply:
Done

Reply:
Can you rename the [bz672510_1.sh](https://github.com/RedHatQE/shadow-utils/blob/dd968fc29862b99325e9cf930793e377781e92f5/multihost_test/Bugzillas/data/bz672510_1.sh#L4) file to a more generic name and call it from both bz#672510 and bz#667593?

Reply:
Done